### PR TITLE
Add a helper for performing editor actions

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
The PR depends on https://github.com/appium/io.appium.settings/pull/18

### CODE REVIEW

File: lib/tools/adb-commands.js [hunk 1]
```diff
@@ -1608,4 +1609,26 @@ methods.screenrecord = function (destination, options = {}) {
...
+ *                                 names are supported: `normal, unspecified, none,
+ *                                 go, search, send, next, done, previous`
+ */
+methods.performEditorAction = async function (action) {
+  log.debug(`Performing editor action: ${action}`);
+  const defaultIME = await this.defaultIME();
+  await this.enableIME(APPIUM_IME);
+  try {
+    await this.setIME(APPIUM_IME);
+    await this.shell('input', 'text', `/${action}/`);
```
Comment (KazuCocoa) [inline lib/tools/adb-commands.js]: `[]` is needed
`await this.shell(['input', 'text', `/${action}/`]);`

Comment (mykola-mokhnach) [reply lib/tools/adb-commands.js]: good catch. thanks

File: lib/tools/adb-commands.js [hunk 2]
```diff
@@ -1608,4 +1609,26 @@ methods.screenrecord = function (destination, options = {}) {
...
+ */
+methods.performEditorAction = async function (action) {
+  log.debug(`Performing editor action: ${action}`);
+  const defaultIME = await this.defaultIME();
+  await this.enableIME(APPIUM_IME);
+  try {
+    await this.setIME(APPIUM_IME);
+    await this.shell(['input', 'text', `/${action}/`]);
+  } finally {
+    await this.setIME(defaultIME);
```
Comment (jlipps) [inline lib/tools/adb-commands.js]: do we want to set it back to the default? what if  the user has already set their own ime? maybe we should get the current IME instead? (could also then save some time if for whatever reason it happens to already be the appium IME)

Comment (mykola-mokhnach) [reply lib/tools/adb-commands.js]: actually the default ime is the currently selected one. It is just the confusing name of the function

**Pull request merged**